tests/language/markdown-language.test.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * @fileoverview Tests for MarkdownLanguage
+ * @author 루밀LuMir(lumirlumir)
+ */
+
+//-----------------------------------------------------------------------------
+// Imports
+//-----------------------------------------------------------------------------
+
+import { MarkdownLanguage } from "../../src/language/markdown-language.js";
+import assert from "node:assert";
+
+//-----------------------------------------------------------------------------
+// Tests
+//-----------------------------------------------------------------------------
+
+describe("MarkdownLanguage", () => {
+	describe("parse()", () => {
+		it("should parse markdown", () => {
+			const language = new MarkdownLanguage();
+			const result = language.parse({
+				body: "# Hello, World!\n\nHello, World!",
+				path: "test.css",
+			});
+
+			assert.strictEqual(result.ok, true);
+			assert.strictEqual(result.ast.type, "root");
+			assert.strictEqual(result.ast.children[0].type, "heading");
+			assert.strictEqual(result.ast.children[1].type, "paragraph");
+		});
+
+		it("should not parse gfm features in commonmark mode", () => {
+			const language = new MarkdownLanguage({ mode: "commonmark" });
+			const result = language.parse({
+				body: "| Column 1 | Column 2 |\n| -------- | -------- |\n| Cell 1   | Cell 2   |",
+				path: "test.md",
+			});
+
+			assert.strictEqual(result.ok, true);
+			// The table should not be parsed and should be recognized as plain text.
+			assert.strictEqual(result.ast.children[0].type === "table", false);
+			assert.strictEqual(
+				result.ast.children[0].type === "paragraph",
+				true,
+			);
```

Comment:
We only need the second assertion here.
```suggestion
			assert.strictEqual(result.ast.children[0].type, "paragraph");
```